tools/sigh.ts

```diff
@@ -432,6 +433,14 @@ function lint(args: string[]): boolean {
   return report.errorCount === 0;
 }
 
+function licenses(): boolean {
```

Comment:
Two things just to check:
- the chat discussion involved thinking that this would be optional or only triggered on `package.json` changes, is it manual for now, does this do that in some way, or do we plan to do that later?
- we also talked about adding a `sigh` task to run instead of an npm command, this PR still adds an npm command, it just also adds a `sigh` task which uses the npm command. That seems unavoidable unless we pull in the tool through some way other than npm, but wanted to mention.

Reply:
I followed the `webpack` pattern here.  It will suffice for now.  There is an API avaialable that we can move to once we are ready to activate the test on changes to package*.json